src/github-graphql-base.js

```diff
@@ -6,7 +6,7 @@ import {
   GITHUB_PR_INFO_QUERY,
   GITHUB_LABEL_INFO_QUERY,
   GITHUB_REACTION_INFO_QUERY
-} from '../script/queries';
+} from '../script/queries/export';
```

Comment:
name the file as `index.js` so we could import without specifying the file name

Reply:
The problem is `pre-fetch.js` is using `index.js`. I am planning to use the new client to rebuild pre-fetching, but not now. So for now, I use `export.js`.